workspaces/redhat-argocd/plugins/argocd/src/components/AppStatus/StatusIcons.tsx

```diff
@@ -15,108 +15,157 @@
  */
 import React from 'react';
 
-import ArrowCircleUpIcon from '@patternfly/react-icons/dist/esm/icons/arrow-alt-circle-up-icon';
-import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
-import CircleNotchIcon from '@patternfly/react-icons/dist/esm/icons/circle-notch-icon';
-import GhostIcon from '@patternfly/react-icons/dist/esm/icons/ghost-icon';
-import HeartBrokenIcon from '@patternfly/react-icons/dist/esm/icons/heart-broken-icon';
-import HeartIcon from '@patternfly/react-icons/dist/esm/icons/heart-icon';
-import PauseCircleIcon from '@patternfly/react-icons/dist/esm/icons/pause-circle-icon';
-import QuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/question-circle-icon';
+import ArrowCircleUpIcon from '@mui/icons-material/ArrowCircleUp';
+import HeartIcon from '@mui/icons-material/FavoriteBorder';
+import HeartBrokenIcon from '@mui/icons-material/HeartBrokenOutlined';
```

Comment:
```suggestion
import HeartBrokenIcon from '@mui/icons-material/HeartBroken';
```

original:
![image](https://github.com/user-attachments/assets/d7dae39a-599b-4d40-8241-0f3443527fb7)


with the current PR changes:

![image](https://github.com/user-attachments/assets/de5c5326-2797-4a8b-8b28-39d0f5f24a6a)

with suggested changes (aligns better with original icon):

![image](https://github.com/user-attachments/assets/3e7c763d-3dfd-479d-ac72-3438c9632b8f)

Reply:
UX prefers outlined icons, I can ask Shiran to review this when she is back

Reply:
sure, I informed Pankaj to take a look.